recipe/meta.yaml

```diff
@@ -1,52 +1,171 @@
 {% set version = "2022.2.1" %}
+{% set number = 0 %}
 
 
 package:
-  name: dask
+  name: dask-split
```

Comment:
Does the package need a name and if it does can the name just be dask?

Reply:
Yeah unfortunately. It also can't duplicate one of the existing subpackages (have struggled with trying to do that before). Typically we just add `-split` here, but we could add anything as long as it doesn't duplicate other packages

That said, agree this is not the most ergonomic thing